packages/expo/src/modules/adoption/reducers.ts

```diff
@@ -0,0 +1,64 @@
+import * as ImagePicker from 'expo-image-picker';
```

Comment:
Missing unit tests for reducers. See my comment above about unifying all reducers in just a single one.